client/console/src/main/java/org/apache/syncope/client/console/panels/ConnObjectListViewPanel.java

```diff
@@ -139,12 +155,26 @@ public void onClick(final AjaxRequestTarget target) {
 
             @Override
             protected Component getValueComponent(final String key, final ConnObjectTO bean) {
-                Optional<AttrTO> attrTO =
-                        bean.getAttrs().stream().filter(object -> object.getSchema().equals(key)).findAny();
-
-                return !attrTO.isPresent() || attrTO.get().getValues().isEmpty()
-                        ? new Label("field", StringUtils.EMPTY)
-                        : new CollectionPanel("field", attrTO.get().getValues());
+                if (StringUtils.equals(key, "Status")) {
```

Comment:
`"Status"` is used twice in this class, please define a class-level constant for it, thanks.